packages/manager/apps/pci-users/src/components/users/RCloneDownloadModal.tsx

```diff
@@ -221,7 +218,7 @@ export default function RCloneDownloadModal({
           slot="actions"
           color={ODS_THEME_COLOR_INTENT.primary}
           onClick={() => download(region, fileType)}
-          {...(isPending || isLoadingDownload ? { disabled: true } : {})}
+          {...(isLoadingDownload ? { disabled: true } : {})}
```

Comment:
```suggestion
          {...(isLoadingDownload && { disabled: true })}
```
nitpick : just a little less verbose